website/layouts/partials/lastmod.html

```diff
@@ -0,0 +1,11 @@
+{{$p := path.Base (path.Split .Path).Dir }} {{$api
+:="https://api.github.com/repos/CodeYourFuture/immersive-go-course/commits" }}
```

Comment:
Similarly, is there a way to avoid reaching out to github for this?

Reply:
I don't see the value of avoiding this. Can you explain more?

Reply:
It's a cause of slowness, failure, and an online dependency. So if you're developing on a plane things won't work, if you're on slow internet it may be slow, or if you're getting rate limited by github (I've hit rate limits when not using an API token before) it may just fail.

That said, I don't actually know when this runs - is this a build-time thing or a run-time thing?

Reply:
So there's a couple of ways I can see immediately to approach this:

1- provide a fallback value for local development, as it's not super necessary for this value to be accurate when locally developing the website
2 - write a script to get the git history out into JSON and call that instead, but that seems like a large undertaking for not very much in this case

Thoughts?

Reply:
I created a fallback value so it defaults to 1970 https://github.com/CodeYourFuture/immersive-go-course/pull/103/commits/f1c61413f95acd32c2314044f5535d4160e3a293